python-sdk/noxfile.py

```diff
@@ -60,7 +60,13 @@ def test(session: nox.Session, airflow) -> None:
     session.run("airflow", "db", "init", env=env)
 
     # Since pytest is not installed in the nox session directly, we need to set `external=true`.
-    session.run("pytest", *session.posargs, env=env, external=True)
+    session.run(
+        "pytest",
+        "-vv",
```

Comment:
@tatiana why do you want to run test with `-vv` option?

Reply:
@pankajastro I always find it useful to have a summary of the tests which are passing/failing while we run the tests - but we can remove it from this PR :)